weave

```diff
@@ -424,6 +429,7 @@ PORT=${WEAVE_PORT:-6783}
 HTTP_ADDR=${WEAVE_HTTP_ADDR:-127.0.0.1:6784}
 PROXY_PORT=12375
 PROXY_CONTAINER_NAME=weaveproxy
+HOST_ROOT=${HOST_ROOT:-/}
```

Comment:
now that HOST_ROOT isn't passed all the time, this should be moved (and perhaps linlined?) to the router launch code.